src/test/rustdoc-js-std/typed-query.js

```diff
@@ -8,5 +8,7 @@ const EXPECTED = {
         { 'path': 'std', 'name': 'eprint' },
         { 'path': 'std', 'name': 'println' },
         { 'path': 'std', 'name': 'eprintln' },
+        { 'path': 'std::pin', 'name': 'pin' },
+        { 'path': 'core::pin', 'name': 'pin' },
```

Comment:
What happened here? Does this test that searching for `macro:print` results in `std::pin::pin`?

Reply:
Yes, rustdoc search is fuzzy, and `pin` seems to be close enough to `print` to then be part of the result; and since _that test suite expects exact results_, I've had to add it. Ideally the test would be done in a non-fuzzy fashion _or_ allow for extra results? I don't know. Can you ping the rustdoc team to have their input on the matter?

Reply:
@rust-lang/rustdoc ping ^

Reply:
I confirm it's a valid update. :)